### PR TITLE
[Launch]: Kubernetes fix image hanging forever

### PR DESCRIPTION
**Pull request by KyleGoyette**
Fixes WB-NNNN
Fixes #NNNN

Description
-----------
What does the PR do?

Testing
-------
How was this PR tested?

Checklist
-------
- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

File: wandb/sdk/launch/runner/kubernetes.py [hunk 1]
```diff
@@ -56,6 +61,19 @@ def get_status(self) -> Status:
             name=self.name, namespace=self.namespace
         )
         status = job_response.status
+        try:
+            pod_status = self.core_api.read_namespaced_pod_log(name=self.pod_names[0], namespace=self.namespace)
+            wandb.termlog(f"{pod_status}")
```
Comment (stephchen) [inline wandb/sdk/launch/runner/kubernetes.py:66]: what's this print for?

File: wandb/sdk/launch/runner/kubernetes.py [hunk 2]
```diff
@@ -56,6 +61,19 @@ def get_status(self) -> Status:
             name=self.name, namespace=self.namespace
         )
         status = job_response.status
+        try:
+            pod_status = self.core_api.read_namespaced_pod_log(name=self.pod_names[0], namespace=self.namespace)
```
Comment (stephchen) [inline wandb/sdk/launch/runner/kubernetes.py:65]: why `read_namespaced_pod_log` and not `read_namespaced_pod_status`?

**Review by stephchen — CHANGES_REQUESTED** [review]

File: wandb/sdk/launch/runner/kubernetes.py (hunk 1 above)
Comment (KyleGoyette) [reply wandb/sdk/launch/runner/kubernetes.py:66]: I'll remove this

File: wandb/sdk/launch/runner/kubernetes.py (hunk 2 above)
Comment (KyleGoyette) [reply wandb/sdk/launch/runner/kubernetes.py:65]: The log issues an error until the container is actually running. While the status would need to be parsed to determine the nature of the failure mode. This is a way to handle any sort of errors (for example image pulling)

**Review by stephchen — APPROVED** [review]

**Pull request merged**